src/components/Groups/CreateGroup/CreateGroupFormSyles.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
```

Comment:
I would remove this file and pass in a background color prop to the styled component. 

Would look something like this https://www.styled-components.com/docs/basics#passed-props